yapapi/__init__.py

```diff
@@ -1,6 +1,8 @@
 """Golem Python API."""
 import logging
 
+__version__: str = "0.2.0"
```

Comment:
PEP 396 - recomends to use `pkg_resources.get_distribution("yapapi").version` is this case.

Reply:
[ to be precise - it doesn't _recommend_ this approach - rather specifies it as an option...  ]

especially that this approach doesn't work from yapapi repo's own code ... 

still - as mentioned in our private conversation - I'm not polarized here...

Reply:
anyway, @prekucki addressed your remark